openlibrary/i18n/__init__.py

```diff
@@ -314,6 +316,8 @@ def __call__(self, string, *args, **kwargs):
         if args:
             value = value % args
         elif kwargs:
+            if isinstance(kwargs.get('count'), Nothing):
```

Comment:
This isn't the right place for this; this just kind of mutes the error. We should stop it at the source where this is being passed in `Nothing`. It looks like it's happening when going to a historical record, eg https://openlibrary.org/books/OL23244503M/Riding_freedom?v=6 , that has a work key that is a redirect. I think I see the fix! Opening a new PR.

Reply:
Also weirdly most of these events seem to be from `bingbot` ; these pages should not be being indexed by search engines!

Reply:
Nevermind, they should be indexed with canonical URLs ; it's correct as is :+1: